sdk/src/main/java/com/hellocustomer/sdk/dialog/HelloCustomerDialogImpl.kt

```diff
@@ -43,13 +46,17 @@ internal class HelloCustomerDialogImpl : DialogFragment(), HelloCustomerDialog {
 
         setupView()
 
-        return Dialog(requireContext()).apply {
+        return AlertDialog.Builder(requireActivity())
```

Comment:
If we don't need this let's roll back the change. We're setting view twice in this implementation